frontend/src/components/navbar/searchbar.tsx

```diff
@@ -1,14 +1,15 @@
 import * as React from "react";
-import SearchByKeyword from "./keywordsearch";
-import DatePickerWithRange from "./date-picker-with-two-range";
+import SearchByKeyword from "./Keywordsearch";
```

Comment:
wrong import name causing it not to be found -> keywordsearch is the correct one
![Screenshot 2024-05-23 at 7 19 06 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/2961c00f-3576-48f1-94c9-83169d6b02bb)